src/config/txpool_size.mlh

```diff
@@ -0,0 +1,3 @@
+(* Note this value needs to be consistent across nodes to prevent spurious bans.
+   see comment in transaction_pool.ml for more details. *)
+[%%define pool_max_size 1024]
```

Comment:
Can you sync with @jkrauska / @es92 / someone who knows to make sure this high enough for the target TPS we want in the shortish-term in the testnet if you haven't already.